biotrainer/utilities/executer.py

```diff
@@ -43,7 +43,11 @@ def parse_config_file_and_execute_run(config_file_path: str):
         config["pretrained_model"] = str(input_file_path / config["pretrained_model"])
 
     # Create output directory (if necessary)
-    output_dir = input_file_path / "output"
+    if "output_dir" in config.keys():
+        output_dir = Path(config["output_dir"])
```

Comment:
I think this will probably only work for absolute paths given in the config file (did you test it?) Might be necessary to do
`output_dir = input_file_path / Path(config["output_dir"]` here as well

Reply:
It works for absolute and relative paths (in this case from the execution point). 


This:
> Might be necessary to do
output_dir = input_file_path / Path(config["output_dir"] here as well

does not make sense for me, since the line `output_dir = input_file_path / “output”` has been deleted. Can you clarify this?

Reply:
The output_dir should be relative to the config file (I think this makes the most sense and is also the default behaviour). I made a small change in order to have that behaviour implemented. Other than that, the PR is very useful :)